examples/make-condor.py

```diff
@@ -0,0 +1,97 @@
+import os
```

Comment:
Thank you @esheldon !

Reply:
I added the script it runs too, but you'll still need to modify paths